js/script.js

```diff
@@ -22,10 +30,87 @@ function closeMenu() {
+          <h4>Professor Emeritus, Korea Advanced Institute of Science and Technology (KAIST)</h4>
+          <div class="red-line"><img src="./images/main-program/red-line.jpg" alt="red line"></div>
+          <p>He developed SDN, the first Internet protocol network in Asia, and led Korea's first dedicated line Internet connection in 1990, opening the Internet era in earnest.</p>
+        </div>
+    </div>
+
+    <div class="speaker-section-div">
+        <div class="speaker-img"><img src="./images/speakers/speaker_03.png" alt="speaker 3"></div>
+        <div class="speaker-details">
+          <h3>Soyoung Roh</h3>
+          <h4>Art Center Butterfly Director, CC Korea</h4>
+          <div class="red-line"><img src="./images/main-program/red-line.jpg" alt="red line"></div>
+          <p>She opened 'Art Center Nabi', Korea's first digital art specialized institution, and has been in office ever since.</p>
+        </div>
+    </div>
+
+    <div class="speaker-section-div">
+        <div class="speaker-img"><img src="./images/speakers/speaker_04.png" alt="speaker 4"></div>
+        <div class="speaker-details">
+          <h3>Julia Reda</h3>
+          <h4>Representative of the Young Pirates of Europe</h4>
+          <div class="red-line"><img src="./images/main-program/red-line.jpg" alt="red line"></div>
+          <p>European integration and youth participation in politics and democracy through online are key concerns, according to a report that could potentially impact changes to the European Union's copyright laws in July.</p>
+        </div>
+    </div>
+
+    <div class="speaker-section-div">
+        <div class="speaker-img"><img src="./images/speakers/speaker_05.png" alt="speaker 5"></div>
+        <div class="speaker-details">
+          <h3>Laila Tretikov</h3>
+          <h4>Wikimedia Foundation Executive Director</h4>
+          <div class="red-line"><img src="./images/main-program/red-line.jpg" alt="red line"></div>
+          <p>Laila Tretikow is executive director of the Wikimedia Foundation, the nonprofit that runs Wikipedia. Wikipedia is available for free every month in 290 languages to more than 100 million people, nearly half of the world's population.</p>
+        </div>
+    </div>
+
+    <div class="speaker-section-div">
+        <div class="speaker-img"><img src="./images/speakers/speaker_06.png" alt="speaker 6"></div>
+        <div class="speaker-details">
+          <h3>Ryan Merkley</h3>
+          <h4>CEO of Creative Commons, former COO of the Mozilla Foundation</h4>
+          <div class="red-line"><img src="./images/main-program/red-line.jpg" alt="red line"></div>
+          <p>He led open source projects at the Mozilla Foundation and joined CC as CEO in 2014. Open government, open source, and other open movements have been actively carried out.</p>
+        </div>
+    </div>
+  </section>
+
+  <button id="more-speakers">MORE&nbsp;&nbsp;<img src="./images/nav/arrow_down.png" alt="down icon"></button>
+`;
```

Comment:
- [ ] Good job adding the `More` button on your mobile design. So, I noticed that on Mobile, only 2 speakers are shown. According to the logic, other speakers should be shown once clicked the `More` button but on your page, it is not the case even if the `MORE` button has been clicked. Make sure you fix that.

Reply:
Thanks but I think it is an optional requirement @tresorsawasawa. Please check the information below: 

![cr2](https://user-images.githubusercontent.com/87370519/204987035-e2dc82ba-6a86-484c-8496-ca83febad58d.jpg)